keras_cv/models/stable_diffusion/text_encoder.py

```diff
@@ -67,7 +79,7 @@ def call(self, inputs):
         residual = x
         x = self.layer_norm2(x)
         x = self.fc1(x)
-        x = x * tf.sigmoid(x * 1.702)  # Quick gelu
+        x = self.gelu(x)
```

Comment:
looks like a behavioral change that will break SD 1.0, right?

Reply:
I didn't found any issue running SD 1.0 in my case.

Reply:
Yeah this shouldn't be an issue, since in the SD 1 case "gelu" is set to `x * tf.sigmoid(x * 1.702)`